### PR TITLE
Update cosmic_settings.ftl

### PR DESCRIPTION
**Pull request by bittin**
Solve Merge conflict with the Swedish translation

### CODE REVIEW

File: i18n/sv/cosmic_settings.ftl [hunk 1]
```diff
@@ -57,22 +57,56 @@ style = Stil
...
+
+font-weight = Vikt
+    .thin = Tunn
+    .extra-light = Extra lätt
+    .light = Lätt
+    .normal = Normal
+    .medium = Medium
+    .semibold = Halvfet
+    .bold = Fet
+    .extra-bold = Extra fet
```
Comment (mmstick) [inline i18n/sv/cosmic_settings.ftl:90]: These do not exist anymore

File: i18n/sv/cosmic_settings.ftl [hunk 2]
```diff
@@ -57,22 +57,56 @@ style = Stil
...
+    .black = Svart
+
+font-style = Stil
+    .normal = Normal
+    .italic = Kursiv
+    .oblique = Sned
```
Comment (mmstick) [inline i18n/sv/cosmic_settings.ftl:96]: These do not exist anymore

File: i18n/sv/cosmic_settings.ftl [hunk 3]
```diff
@@ -57,22 +57,56 @@ style = Stil
...
+
+font-stretch = Sträcka
+    .condensed = Förtätad
+    .normal = Normal
+    .expanded = Expanderad
```
Comment (mmstick) [inline i18n/sv/cosmic_settings.ftl:101]: These do not exist anymore

File: i18n/sv/cosmic_settings.ftl [hunk 4]
```diff
@@ -57,22 +57,56 @@ style = Stil
...
+
 ## Testinställningar
 
 experimental-settings = Testinställningar
 icons-and-toolkit = Utformning av ikoner och verktyg
 interface-font = Systemets teckensnitt
 monospace-font = Teckensnitt med jämna mellanrum
+>>>>>>> master
```
Comment (mmstick) [inline i18n/sv/cosmic_settings.ftl:109]: Merge conflict syntax left over

File: i18n/sv/cosmic_settings.ftl [hunk 5]
```diff
@@ -57,22 +57,56 @@ style = Stil
...
     .active-hint = Storlek på aktivt fönsterhinting
     .gaps = Springor runt kaklade fönster
 
+
+### Appearance: Teckensnitt
+
+font-config = Teckensnittskonfiguration
+interface-font = System teckensnitt
+monospace-font = Monospace teckensnitt
+font-family = Teckensnittsfamilj
```
Comment (mmstick) [inline i18n/sv/cosmic_settings.ftl:80]: Not used

File: i18n/sv/cosmic_settings.ftl [hunk 6]
```diff
@@ -57,22 +57,56 @@ style = Stil
...
 window-management-appearance = Fönsterhantering
     .active-hint = Storlek på aktivt fönsterhinting
     .gaps = Springor runt kaklade fönster
 
+
+### Appearance: Teckensnitt
+
+font-config = Teckensnittskonfiguration
+interface-font = System teckensnitt
+monospace-font = Monospace teckensnitt
```
Comment (mmstick) [inline i18n/sv/cosmic_settings.ftl:79]: Duplicate variable

File: i18n/sv/cosmic_settings.ftl [hunk 7]
```diff
@@ -57,22 +57,56 @@ style = Stil
...
 window-management = Fönsterhantering
 window-management-appearance = Fönsterhantering
     .active-hint = Storlek på aktivt fönsterhinting
     .gaps = Springor runt kaklade fönster
 
+
+### Appearance: Teckensnitt
+
+font-config = Teckensnittskonfiguration
+interface-font = System teckensnitt
```
Comment (mmstick) [inline i18n/sv/cosmic_settings.ftl:78]: Duplicate variable

**Review by mmstick — CHANGES_REQUESTED** [review]

**Comment by bittin** [comment]
Sorry, should look at this properly when i have more time, would be good with https://weblate.org/en-gb/ 
https://hosted.weblate.org/ or a translation system instead of doing everything manually tbh

**Comment by bittin** [comment]
to much mess see #725 instead for now

**Pull request closed without merging**